projects/docs/src/app/docs-wrapper/docs-wrapper.component.scss

```diff
@@ -2,8 +2,6 @@
 
 .docs-wrapper {
     width: 100vw;
-    height: 100vh;
-    overflow: hidden;
```

Comment:
I believe we need these right? Otherwise, scrolling throw the documentation will not have a good alignment with the side menu

Reply:
No we dont need it, its child `.content-wrapper` have:
```
height: 100vh;
overflow-y: auto;
```